wordpress-mobile/WordPress-Android
# Move image editor library inside libs

This PR
- Moves `ImageEditor` library inside `libs` folder.
- Includes `ImageEditor` library in `WPAndroid` from `libs` folder.

(Also updates .gitignore and adds gradle wrapper files similar to other libraries inside libs)

**To test:**
- `ImageEditor` files git history is not lost.
(It seems [we cannot see this history in GitHub ](https://git.io/Jvyrt) but we can try [this chrome extension](https://chrome.google.com/webstore/detail/follow-for-github/agalokjhnhheienloigiaoohgmjdpned) if we really want to.)
- `WPAndroid` compiles and `ImageEditor` can be launched properly.

PR submission checklist:
